cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -238,14 +258,21 @@ func (s *Server) pullChartTarball(ctx context.Context, packageRef *corev1.Availa
 		Resource: fluxHelmCharts,
 	}
 
+	// TODO: use the namespace from the request
```

Comment:
Is this a TODO only because it's a pain to construct a request currently (as we're not yet mapping the namespace to the URL)? Otherwise why not just use request.Namespace or whatever the field is?

Reply:
Nothing hard about it. Just didn't get to it yet :-) Will happen soon